### PR TITLE
Added recipe and license for package mockextras

### PR DESCRIPTION
**Pull request by trendelkampschroer**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)

### CODE REVIEW

File: recipes/mockextras/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  license_file: LICENSE.txt
+  summary: Extensions to the mock library
+  description: |
+    "The mockextras library is designed to be used with the
+    unittest.mock library in python 3 or the mock backport of this
+    (http://www.voidspace.org.uk/python/mock/) in python 2. The mockextras
+    library adds a number of features that are found in other mocking
+    libraries namely: 
+       * a fluent API for the configuration of stubs
+       *stubs
```
Comment (scopatz) [inline recipes/mockextras/meta.yaml]: Seems like there is a space missing

File: recipes/mockextras/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - mockextras
+
+about:
+  home: https://github.com/manahl/mockextras
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Extensions to the mock library
+  description: |
+    "The mockextras library is designed to be used with the
```
Comment (scopatz) [inline recipes/mockextras/meta.yaml]: You don't need quotes for this block

File: recipes/mockextras/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pytest
+    - coveralls
+    - pytest-cov
+    - coverage
+    - mock >=0.8.0  # [py<33]
```
Comment (scopatz) [inline recipes/mockextras/meta.yaml]: Do you actually need Python 2.7?  It would be much bettrer if this could be noarch: python

Comment (trendelkampschroer) [reply recipes/mockextras/meta.yaml]: Yes, this is a dependency for the `arctic` library (https://pypi.org/project/arctic/), which still supports `python 2.7`. In order to release a healthy conda package for arctic we need this package to be compatible with `python 2.7` also.

File: recipes/mockextras/meta.yaml (hunk 1 above)
Comment (trendelkampschroer) [reply recipes/mockextras/meta.yaml]: Done!

File: recipes/mockextras/meta.yaml (hunk 2 above)
Comment (trendelkampschroer) [reply recipes/mockextras/meta.yaml]: Done!

**Comment by trendelkampschroer** [comment]
@scopatz Thanks a lot for your review. If this looks good to you, then I'd ask you to close this at your earliest convenience. We need this feedstock in order to update another recipe which depends on mockextras becoming available on conda-forge.

**Comment by trendelkampschroer** [comment]
@ocefpaf: Could you maybe take a look at this I haven't received a notification from @scopatz yet?

**Comment by trendelkampschroer** [comment]
Thanks a lot @beckermr

**Pull request merged**